.github/workflows/cri_test.yml

```diff
@@ -26,7 +26,7 @@ jobs:
 
     - name: Host Info
       env:
-          GITHUB_RUN_ID: ${{ github.run_id }}-${{ github.run_number }}
+          GITHUB_RUN_ID: ${{ github.run_id }}
```

Comment:
please explain these changes in the PR's description

Reply:
`run_number` is turning out to be empty. Hence, removed.